build.gradle

```diff
@@ -123,6 +123,7 @@ dependencies {
 
   compile group: 'io.github.openfeign', name: 'feign-httpclient', version: '9.5.0'
   compile group: 'uk.gov.hmcts.reform', name: 'document-management-client', version: '1.0.2'
+  compile group: 'uk.gov.hmcts.reform', name: 'service-authorisation-access-client', version: '0.0.2'
```

Comment:
Why is this version 0.0.2. It does not give me confidence. When `service-authorisation-access-client` was developed it should have got much higher version.

Reply:
Rafal k suggested to start release 0.0.1 , now we have 0.0.2 after a fix by Tim. Rafal also want to add some more api in this for their use like 'details'